docs/gettsim_developer/hh_concepts.md

```diff
@@ -0,0 +1,346 @@
+  (it follows the SGB II definition).
+- Potentially, `bg_id` could be used as the aggregation unit.
+
+#### Pointers
+
+- `p_id_einstandspartner`, `p_id_elternteil_1`, `p_id_elternteil_2` (exogenous)
+
+### SGB XII (Eingliederungshilfe für Menschen mit Behinderung)
+
+#### Description
+
+Government expenditures: 20 Mrd €
+
+- Not implemented.
+
+### SGB XII (Hilfe zur Pflege)
+
+#### Description
+
+Government expenditures: 4 Mrd €
+
+- Not implemented.
+
+### Interactions between SGB II / SGB XII
+
+- Partnerships where one partner falls under SGB II and one under SGB XII -> Whole
+  household falls under SGB II
+- Bedarfsgemeinschaft with child under 25 falling under SGB II -> ?
+
+## Wohngeld
+
+#### Description
+
+- Household (§5 WoGG)
+  - Applicant
+  - Spouse / registered partner / life partner (as long as in same household)
+  - Parents (incl. step-, foster- and in-laws) (as long as in same household)
+  - Children (incl. step-, foster- and adopted children) (as long as in same household)
+  - "Einstandspartner": non-relatives living in the same household and sharing
+    responsibility for each other (e.g. couples that are not married or in a civil
+    union) (as long as in same household)
+    - A community of responsibility or joint liability is regulated in § 7 Abs. 3a SGB
+      II according to which at least one of the following conditions must be met
+      - if partners live together for longer than 1 year
+      - if partners live together with a child
+      - if children or relatives are cared for or looked after in the household (not
+        only occasionally)
+      - if partners are mutually authorized to dispose of income and assets
+    - The prerequisite for all household members is that they live with the applicant
+      live in a shared apartment or house and are also registered there.
+  - Other relatives
+- wohngeldrechtlicher Teilhaushalt: Level at which the transfer is paid out. A household
+  consists of at most one wohngeldrechtlicher Teilhaushalt with positive Wohngeld
+  transfer and may consist of several additional Bedarfsgemeinschaften.
+  - Wohngeld has priority over SGB II transfers. The Vorrangprüfung decides which
+    Bedarfsgemeinschaft gets Wohngeld and which gets SGB II.
+  - Households that consist of at least one Bedarfsgemeinschaft and one
+    wohngeldrechtlicher Teilhaushalt are called "Mischhaushalte".
+
+#### Aggregation unit
+
+- `wthh_id` (endogenous)
+
+#### Pointers
+
+- `hh_id`, `p_id_einstandspartner`, `p_id_elternteil_1`, `p_id_elternteil_2` (exogenous)
+
+### Kinderwohngeld
+
+#### Description
+
+- Children that are part of the Bedarfsgemeinschaft of their parents can receive
+  Wohngeld to drop out of the Bedarfsgemeinschaft ("Kinderwohngeld")
+- Only possible if the child can cover its needs with the Wohngeld transfer
+- Does not happen automatically, but upon request by the parents
+- Not implemented yet.
+
+## Modelling groupings
```

Comment:
I am not sure about this section. Maybe label as some random notes?

Reply:
I changed the title to "Limitations" and tailored the content a bit.